src/test/rustdoc/empty-section.rs

```diff
@@ -8,3 +8,4 @@ pub struct Foo;
 // @!has - 'Auto Trait Implementations'
 impl !Send for Foo {}
 impl !Sync for Foo {}
+impl !std::marker::Unpin for Foo {}
```

Comment:
NB: this test still fails, because it is not possible to write a negative implementation for the private `Freeze` trait.